thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeExecutor.java

```diff
@@ -0,0 +1,72 @@
+package com.linkedin.thirdeye.anomaly.merge;
+
+import com.linkedin.thirdeye.api.dto.GroupByKey;
+import com.linkedin.thirdeye.api.dto.GroupByRow;
+import com.linkedin.thirdeye.client.pinot.PinotThirdEyeClient;
+import com.linkedin.thirdeye.db.dao.AnomalyMergedResultDAO;
+import com.linkedin.thirdeye.db.dao.AnomalyResultDAO;
+import com.linkedin.thirdeye.db.entity.AnomalyMergedResult;
+import com.linkedin.thirdeye.db.entity.AnomalyResult;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * finds raw anomalies grouped by a strategy and merges them with an existing (in the same group) or
+ * new {@link com.linkedin.thirdeye.db.entity.AnomalyMergedResult}
+ */
+public class AnomalyMergeExecutor {
```

Comment:
I have added todos in updateMergedResults method. A job can call this method periodically and this will take care of updating raw results as per specified mergeConfig.mergeStrategy. 
This is WIP.